client/src/plugins/user-journey-statistics/event-handlers/TabEventHandler.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+import { getEngineProfile } from '../Util';
+
+const types = {
+  BPMN: 'bpmn',
+  DMN: 'dmn',
+  CMMN: 'cmmn',
+  FORM: 'form'
+};
+export default class TabEventHandler {
+  constructor(props) {
+
+    const {
+      subscribe,
+      track
+    } = props;
+
+    this.track = track;
+
+    this.subscribeToTabEvents(subscribe);
+  }
+
+  subscribeToTabEvents = (subscribe) => {
+
+    subscribe('bpmn.modeler.created', async (event) => {
+      await this.trackDiagramOpened(types.BPMN, event);
+    });
+
+    subscribe('dmn.modeler.created', async (event) => {
+      await this.trackDiagramOpened(types.DMN, event);
+    });
+
+    subscribe('form.modeler.created', async (event) => {
+
+      const {
+        tab
+      } = event;
+
+      const {
+        file
+      } = tab;
+
+      const {
+        contents
+      } = file;
+
+      let engineProfile;
+
+      if (contents) {
+        try {
+          const schema = (JSON.parse(contents));
+
+          const {
+            executionPlatform,
+            executionPlatformVersion
+          } = schema;
+
+          if (executionPlatform) {
+
+            engineProfile = executionPlatformVersion ?
+              { executionPlatform: executionPlatform, executionPlatformVersion: executionPlatformVersion }
+              : { executionPlatform: executionPlatform };
+          }
+
+        } catch (error) {
+          return;
+        }
+      }
+
+      await this.trackDiagramOpened(types.FORM, event, engineProfile);
+
+    });
+
+    subscribe('tab.closed', (e) => {
+      this.track('diagram:closed');
+    });
+  }
+
+  trackDiagramOpened = async (diagramType, { tab }, engineProfile) => {
```

Comment:
The implementation is not in sync with the specification. 

E.g. [here](https://github.com/camunda/camunda-modeler/issues/2934#issue-1216112738) we state that `diagram:opened` only has a `diagramType` property, where the implementation provides more / different stuff.

To allow for easy review, could you please ensure the events specified are up to date with the implementation?